udata/core/dataset/views.py

```diff
@@ -82,6 +82,9 @@ def get_context(self):
                 abort(404)
             elif self.dataset.deleted:
                 abort(410)
+        context['site_url'] = 'http{ssl}://{site_id}'.format(
```

Comment:
or url_for('site.home_redirect', _external=True) (may be used directly into template).